solutions/fscloud/README.md

```diff
@@ -0,0 +1,9 @@
+# IBM Elastic search fscloud on IBM Cloud
+
+This architecture creates an IBM Elastic search fscloud instance in IBM Cloud  environment and supports provisioning the following resources:
+
+- A resource group, if one is not passed in.
+- An IBM Elastic search fscloud instance on IBM Cloud
+- A IBM Elastic search database
+
+![fscloud-elastic-search](../../reference-architecture/fscloud_elastic_search.svg)
```

Comment:
It should be the complete URL else it will break after onboarding to catalog.

Reply:
cloudant uses it same way - https://github.com/terraform-ibm-modules/terraform-ibm-cloudant/blob/main/solutions/dedicated/README.md?plain=1#L9